vmware-puppetdb-trueup/compare-vmware-to-puppetdb.ps1

```diff
@@ -3,19 +3,31 @@ param (
     [String]$VMwareUserName,
     [SecureString]$VMwarePassword,
     [String]$VMwareCluster = 'operations2',
-    [String]$PupppetServer = 'https://puppet.ops.puppetlabs.net',
-    [Sstring]$Token = $(Get-Content -Path "$HOME/.puppetlabs/token")
+    [String]$PuppetServer = 'https://puppet.ops.puppetlabs.net',
+    [String]$Token = $(Get-Content -Path "$HOME/.puppetlabs/token" -ErrorAction SilentlyContinue),
 )
 
-$pdb_api = "${PupppetServer}:8081/pdb/query/v4"
+if ($PuppetServer -match 'https://') {
+    $port = 8081
+    $SkipCertificateCheck = true
+} else {
+    $port = 8080
+    $SkipCertificateCheck = false
+}
+$pdb_api = "${PuppetServer}:${port}/pdb/query/v4"
 
 $headers = @{
     'Content-Type'     = 'application/json'
```

Comment:
Since other changes are needed anyhow, please remove the now superfluous white space here.
```suggestion
    'Content-Type' = 'application/json'
```